components/SignUpForm.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
```

Comment:
I'm not really sure to use ```redux-form```. Because I think that every new third-party library is creating more complexity to understand the boilerplate. What do you think?